client/src/BookingPage.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import Grid from '@mui/material/Grid';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import IconButton from '@mui/material/IconButton';
+import AppBar from '@mui/material/AppBar';
+import AccountBoxIcon from '@mui/icons-material/AccountBox';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import StaticDatePicker from '@mui/lab/StaticDatePicker';
+import TextField from '@mui/material/TextField';
+import TimePicker from '@mui/lab/TimePicker';
+
+
+
+function Booking_page(){
+    const [dateValue, setDateValue] = React.useState<Date | null>(new Date());
```

Comment:
Use `useState` instead of `React.useState`, this requires the import to be changed to `import React, { useState } from "react"`